src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -1,37 +1,36 @@
 ---
-group: release-notes
-title: Backward Incompatible Changes
+group: cloud-guide
+title: Backward incompatible changes
 ---
 
-Use the following information to learn about backward incompatible changes that require you to adjust Cloud configuration and processes for existing Cloud projects when you upgrade to the latest release of the {{site.data.var.ct}} package or other {{site.data.var.csuite}} packages.
+Use the following information to learn about backward incompatible changes that might require you to adjust Cloud configuration and processes for existing Cloud projects when you upgrade to the latest release of the {{site.data.var.ct}} package or other {{site.data.var.csuite}} packages.
 
 ### Service version requirement changes
 
-**PHP minimum version requirement**–In {{ site.data.var.ct }} v2002.1.0 and later, the minimum php version requirement is 7.1.3+. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file. 
+We changed the minimum PHP minimum requirement from 7.1.0 to 7.1.3+ in `{{ site.data.var.ct }}` v2002.1.0 and later. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file.
 
-### Environment variable changes
+### Environment configuration changes
```

Comment:
Not clear what you mean -- the title should be more specific?  Do you have a suggestion?  So far, the table contains variable changes and removal of the build_options.ini file.  I had in mind that the table can contain any changes related to environment configuration. I can review the release notes to see what other types of changes have occurred in past releases.

Reply:
Oh, yeah. Since it was limited to backward incompatible changes, I thought maybe this would be a list of deprecated things. I could be misunderstanding. My suggestion would be something like Deprecated environment configurations...or similar.